spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -185,6 +186,26 @@ public long executeTask(String taskName, Map<String, String> taskDeploymentPrope
 
 		TaskExecution taskExecution = taskExecutionRepositoryService.createTaskExecution(taskName);
 
+		if (StringUtils.hasText(taskExecutionInformation.getTaskDefinition().getDslText())
+				&& TaskServiceUtils.isComposedTaskDefinition(taskExecutionInformation.getTaskDefinition().getDslText())) {
+			boolean containsAccessToken = false;
+
+			for (String commandLineArg : commandLineArgs) {
+				if (commandLineArg.startsWith("--dataflow-server-access-token")) {
```

Comment:
We need to pass this is in as a property.  It being used as a command-line arg means the token will be stored in the database.

Reply:
Made the change